examples/cpp/metadata/greeter_server.cc

```diff
@@ -44,7 +44,7 @@ class GreeterServiceImpl final : public Greeter::Service {
 
     // Get the client's initial metadata
     std::cout << "Client metadata: " << std::endl;
-    const std::multimap<grpc::string_ref, grpc::string_ref> metadata = context->client_metadata();
+    const std::multimap<std::string_ref, std::string_ref> metadata = context->client_metadata();
```

Comment:
I think `string_ref` needs to stay `grpc` :)

Reply:
Oh my bad! I'm fixing it.